keyboards/orthodox/keymaps/shaymdev/rules.mk

```diff
@@ -0,0 +1,7 @@
+RGBLIGHT_ENABLE = yes
+VELOCIKEY_ENABLE = yes
+KEY_OVERRIDE_ENABLE = yes
+COMBO_ENABLE = yes
+LTO_ENABLE = yes #Link Time Optimization for smaller file size
+CONSOLE_ENABLE = no
+MOUSEKEY_ENABLE = no
```

Comment:
```suggestion
MOUSEKEY_ENABLE = no
RGBLIGHT_ENABLE = yes
VELOCIKEY_ENABLE = yes
KEY_OVERRIDE_ENABLE = yes
COMBO_ENABLE = yes
LTO_ENABLE = yes
```

Reply:
Thanks for looking at my PR! It appears you reordered the settings and removed the console_enable and the comment on LTO. Am I seeing that right? Is there a reason for those changes? I'm just trying to learn the so I can make things better on my future PRs.

Reply:
Console seems to already be disabled at the keyboard level, so there's no need to specify it here.

Reply:
Any reason to remove my comment on the LTO? 

How about the order of the lines? Is there a convention for that or something?